content/en/docs/pluggable-components/logging.md

```diff
@@ -0,0 +1,149 @@
+
+{{</ notice >}}
+
+2. In this file, navigate to `logging` and change `false` to `true` for `enabled`. Save the file after you finish.
+
+```bash
+logging:
+    enabled: true # Change "false" to "true"
+```
+
+3. Create a cluster using the configuration file:
+
+```bash
+./kk create cluster -f config-sample.yaml
+```
+
+### **Installing on Kubernetes**
+
+When you install KubeSphere on Kubernetes, you need to download the file [cluster-configuration.yaml](https://raw.githubusercontent.com/kubesphere/ks-installer/master/deploy/cluster-configuration.yaml) for cluster setting. If you want to install Logging, do not use `kubectl apply -f` directly for this file.
+
+1. In the tutorial of [Installing KubeSphere on Kubernetes](https://kubesphere-v3.netlify.app/docs/installing-on-kubernetes/introduction/overview/), you execute `kubectl apply -f` first for the file [kubesphere-installer.yaml](https://raw.githubusercontent.com/kubesphere/ks-installer/master/deploy/kubesphere-installer.yaml). After that, to enable Logging, create a local file cluster-configuration.yaml.
+
+```bash
+vi cluster-configuration.yaml
+```
+
+2. Copy all the content in the file [cluster-configuration.yaml](https://raw.githubusercontent.com/kubesphere/ks-installer/master/deploy/cluster-configuration.yaml) and paste it to the local file just created.
+3. In this local cluster-configuration.yaml file, navigate to `logging` and enable Logging by changing `false` to `true` for `enabled`. Save the file after you finish.
+
+```bash
+logging:
+    enabled: true # Change "false" to "true"
+```
+
+4. Execute the following command to start installation:
+
+```bash
+kubectl apply -f cluster-configuration.yaml
+```
+
+## Enable Logging after Installation
+
+1. Log in the console as `admin`. Click **Platform** at the top left corner and select **Clusters Management**.
+
+![clusters-management](https://ap3.qingstor.com/kubesphere-website/docs/20200828111130.png)
+
+2. Click **CRDs** and enter `clusterconfiguration` in the search bar. Click the result to view its detailed page.
+
+{{< notice info >}}
+
+A Custom Resource Definition (CRD) allows users to create a new type of resources without adding another API server. They can use these resources like any other native Kubernetes objects.
+
+{{</ notice >}}
+
+3. In **Resource List**, click the three dots on the right of `ks-installer` and select **Edit YAML**.
+
+![edit-yaml](https://ap3.qingstor.com/kubesphere-website/docs/20200827182002.png)
+
+4. In this yaml file, navigate to `logging` and change `false` to `true` for `enabled`. After you finish, click **Update** at the bottom right corner to save the configuration.
```

Comment:
Same as above, add a single notice for the external ES if users are preparing for the production environment.

Reply:
For Logging, Events, and Auditing system, as soon as users enable any of the three components, the installer will install (or rely on) an Elasticsearch. The notice above should be also added in the other two guides (KubeSphere Events System and KubeSphere Auditing System).